### PR TITLE
adds Text2D feature, using a canvas2D

### PR DESCRIPTION
**Pull request by jtorresfabra**
@cedricpinson  you can flag it as Prototype. 	Even though it is working so far.

### CODE REVIEW

File: sources/osgText/Text.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+                this._textX = this._canvas.width;
+                this._context.textBaseline = 'bottom';
+                this._textY = this._canvas.height;
+                break;
+            }
+        },
+
+        _nextPowerOfTwo: function ( value, pow ) {
+            pow = pow || 1;
+            while ( pow < value ) {
```
Comment (cedricpinson) [inline sources/osgText/Text.js]: probably there is a formula for this :)

**Comment by cedricpinson** [comment]
Very cool to have osgText :+1: :)

**Comment by jtorresfabra** [comment]
@cedricpinson  I think it's ready to be reviewed.

**Comment by jtorresfabra** [comment]
I just added the 'Attribute' Suffix because in OSG the class Billboard exist, but it is derived from Geode. I mean, it is not an attribute in OSG. I thought about implement the Billboard Class using Billboard attribute later. 

Also I'm unsure about adding it to osgText, because Billboard is in osg library. 

What do you think?

**Comment by cedricpinson** [comment]
you are right, so did you try with something else like models instead of just textured quad

**Comment by jtorresfabra** [comment]
It does work with models too, but implementing the complete Billboard class would need some work, as implementing pivot points, axis, etc.

**Pull request merged**